tests/integration/conftest.py

```diff
@@ -271,6 +271,20 @@ def with_table_mapping_rule(
     def with_table_mapping_rules(self, rules):
         self.installation.save(rules, filename=TableMapping.FILENAME)
 
+    # For Tables in mounts, we cannot create tables in adls:/ directly, we have to workaround by injecting a mocked crawler
+    def with_static_table_crawler(self, table_crawler: StaticTablesCrawler):
+        for table in table_crawler.snapshot():
+            self._tables.append(
+                TableInfo(
+                    catalog_name=table.catalog,
+                    schema_name=table.database,
+                    name=table.name,
+                    storage_location=table.location,
+                    table_type=TableType(table.object_type),
+                    data_source_format=DataSourceFormat(table.table_format),
+                )
+            )
```

Comment:
```suggestion
```
use `runtime_ctx.make_table`, don't break the class that used by literally every test.